### DISCUSSION TITLE
Sponsors-only actions

### DISCUSSION THREAD

**Discussion by reeseschultz** [community/community#12563] [Actions] [answered]
(Cross-posting from [here](https://github.community/t/github-action-action-in-private-repository/16063/95?u=reeseschultz), as @kaietc recommended.)

Really curious about something: I want to offer a [sponsors-only](https://github.blog/2022-02-02-new-sponsors-only-repositories-custom-amounts-and-more/) GitHub Action, which must be necessarily private. Can any staff speak to this possibility, or consider adding this to your roadmap? The ability to sponsor actions seems like a game-changing feature, because we all know that CI/CD is so rife with maintenance and support.

There are workarounds that may be suitable for said use case, but official support would be fantastic.

**Comment by ethomson** [comment, marked as answer]
Hi @reeseschultz - this is an interesting request, indeed! At the moment, we don't have this on the roadmap, but I'll chat with the team about it and mention it to the sponsors team. It seems very clever.

Thanks much for taking the time to leave feedback.
